src/test/java/com/commercetools/sync/products/helpers/VariantReferenceResolverTest.java

```diff
@@ -26,461 +24,418 @@
-        final CustomFieldsDraft customFieldsDraft = CustomFieldsDraft
-            .ofTypeIdAndJson("customTypeId", new HashMap<>());
-
-        final AssetDraft assetDraft = AssetDraftBuilder.of(emptyList(), ofEnglish("assetName"))
-                                                       .custom(customFieldsDraft)
-                                                       .build();
+    void resolveReferences_WithTextAttribute_ShouldReturnEqualDraft() {
+        // preparation
+        final AttributeDraft textAttribute = AttributeDraft.of("attributeName", "attributeValue");
+        final ProductVariantDraft productVariantDraft = ProductVariantDraftBuilder
+            .of()
+            .attributes(textAttribute)
+            .build();
 
-        final ProductVariantDraftBuilder productVariantDraftBuilder =
-            ProductVariantDraftBuilder.of().assets(singletonList(assetDraft));
+        // test
+        final ProductVariantDraft resolvedAttributeDraft =
+            referenceResolver.resolveReferences(productVariantDraft)
+                             .toCompletableFuture()
+                             .join();
+        // assertions
+        assertThat(resolvedAttributeDraft).isEqualTo(productVariantDraft);
+    }
 
-        final ProductVariantDraftBuilder resolvedBuilder = referenceResolver
-            .resolveAssetsReferences(productVariantDraftBuilder).toCompletableFuture().join();
+    @Test
+    void resolveReferences_WithNullAttributeValue_ShouldReturnEqualDraft() {
+        // preparation
+        final AttributeDraft textAttribute = AttributeDraft.of("attributeName", "attributeValue");
+        final AttributeDraft nullAttributeValue = AttributeDraft.of("attributeName", null);
+        final ProductVariantDraft productVariantDraft = ProductVariantDraftBuilder
+            .of()
+            .attributes(textAttribute, nullAttributeValue)
+            .build();
 
-        final List<AssetDraft> resolvedBuilderAssets = resolvedBuilder.getAssets();
-        assertThat(resolvedBuilderAssets).isNotEmpty();
-        final AssetDraft resolvedAssetDraft = resolvedBuilderAssets.get(0);
-        assertThat(resolvedAssetDraft).isNotNull();
-        assertThat(resolvedAssetDraft.getCustom()).isNotNull();
-        assertThat(resolvedAssetDraft.getCustom().getType().getId()).isEqualTo("typeId");
+        // test
+        final ProductVariantDraft resolvedAttributeDraft =
+            referenceResolver.resolveReferences(productVariantDraft)
+                             .toCompletableFuture()
+                             .join();
+        // assertions
+        assertThat(resolvedAttributeDraft).isEqualTo(productVariantDraft);
     }
 
     @Test
-    void resolvePricesReferences_WithNullPrices_ShouldNotResolvePrices() {
-        final ProductVariantDraftBuilder productVariantDraftBuilder = ProductVariantDraftBuilder.of();
+    void resolveReferences_WithEmptyValue_ShouldReturnEqualDraft() {
```

Comment:
WithEmptyAttributeValue ?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/442/commits/a929072d302551c17a52f26274425e1221f58c41

Reply:
shouldReturnEqualDraft 🤔

Reply:
because here there no references, so saying `shouldResolveReferences` would be a bit weird :)